app/views/deploys/_deploy.html.erb

```diff
@@ -1,38 +1,36 @@
-<li class="deploy" id="deploy-<%= deploy.id %>" data-task-status="<%= deploy.status %>">
-  <div class="status <%= deploy.status %>" data-tooltip="<%= deploy.status.capitalize %>">
-    <%= link_to stack_deploy_path(@stack, deploy) do %>
-      <i><%= deploy.status %></i>
-    <% end %>
+<li class="commit" id="deploy-<%= deploy.id %>" data-task-status="<%= deploy.status %>">
+  <div class="commit-author">
```

Comment:
Is this different from https://github.com/Shopify/shipit-engine/pull/381/files#diff-639adb7d95b6704dc51bfc498cc29256R2 ? Otherwise it could be a partial